src/test/java/com/google/sps/servlets/ContestServletTest.java

```diff
@@ -0,0 +1,131 @@
+package com.google.sps.servlets;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalURLFetchServiceTestConfig;
+import com.google.common.collect.ImmutableSet;
+import java.io.PrintWriter;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+public class ContestServletTest {
+  private static final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig(), new LocalURLFetchServiceTestConfig());
+
+  @Mock HttpServletRequest httpServletRequest;
+  @Mock HttpServletResponse httpServletResponse;
+  @Mock PrintWriter printWriter;
+
```

Comment:
Make the Set an ImmutableSet<Long> instead. 

Generally speaking, later callers of your variable don't need to care if the thing is a TreeSet or HashSet or what, they just need to know it's a Set. 

But Immutable is different - later callers should really know that the thing is Immutable (so that they don't try and mutate it).

Reply:
done